06_gpu_and_ml/gpu_packing.py

```diff
@@ -1,3 +1,6 @@
+# ---
+# mypy: ignore-errors
```

Comment:
why is this needed?

Reply:
Posted in [Slack](https://modal-com.slack.com/archives/C056CGAANRM/p1744770841150439).

Generally dubious of the utility of mypy for the examples -- we can't type-check the logic inside of Functions -- so very quick to turn it off when it gets in the way.